src/validation.cpp

```diff
@@ -2820,10 +2820,10 @@ bool FindUndoPos(CValidationState &state, int nFile, CDiskBlockPos &pos, unsigne
     return true;
 }
 
-bool CheckBlockHeader(const CBlockHeader& block, CValidationState& state, const Consensus::Params& consensusParams, bool fCheckPOW)
```

Comment:
Please rename this while you're at it. CheckBlockHeader is a terrible name if basically just compares 2 values :)